get_de_genes.R

```diff
@@ -0,0 +1,23 @@
+
+# function to extract significant differentially expressed genes
+# dds is a deseq object
+# col is the column of interest
+# cont1 is the first contrast element
+# cont2 is the second contrast element
+get_de_genes <- function(dds, col, cont1, cont2) {
```

Comment:
I think the libraries used in each code snippet should also documented (either in the code or in the README). Because if I try to adapt this code in my analysis, I am not going to run it successful without loading the correct library.

Reply:
Fixed - I documented the libraries required for each code snippet/function in the code.